src/pages/DetailPage.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+import Heading from "../components/Header";
+import Button from "../components/Button";
+import DividingLine from "../components/DividingLine";
+
+DetailPage.propTypes = {
+  concert: PropTypes.object,
+  onNavigate: PropTypes.func,
+  pageName: PropTypes.string,
+};
+
+export default function DetailPage({ concert, onNavigate, pageName }) {
+  return (
+    <div>
+      <Heading pageName={pageName} /> <DividingLine />
+      <Wrapper>
+        <ImageContainer src={concert.image} alt="" />
```

Comment:
Wouldn't call it `Container` as it's only an image. `Image` should be sufficient.